src/operators/plus_equal.cpp

```diff
@@ -31,6 +31,68 @@
 #include "../bigint.hpp"
 
 namespace libbig {
+  largeInt &operator += (largeInt a) {
+        largeInt res;
+        string s1 = this.number;
+        string s2 = a.number;
+        res = addLargeInt(s1, s2);
+        return res;
+    }
+    largeInt &operator += (int a) {
+        largeInt res;
+        string s1 = this.number;
+        string s2 = std::to_string(a);
+        res = addLargeInt(s1, s2);
+        return res;
+    }
+    largeInt &operator += (int64_t a) {
+        largeInt res;
+        string s1 = this.number;
+        string s2 = std::to_string(a);
+        res = addLargeInt(s1, s2);
+        return res;
+    }
+    largeInt &operator += (long long a) {
+        largeInt res;
+        string s1 = this.number;
+        string s2 = std::to_string(a);
+        res = addLargeInt(s1, s2);
+        return res;
+    }
 
+    largeInt addLargeInt(string s1, string s2) {
+        largeInt temp;
+        string tempS = "";
```

Comment:
Codacy found an issue: [Variable 'tempS' is assigned a value that is never used.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285288)